hazelcast/reactor.py

```diff
@@ -22,14 +23,11 @@
 
 try:
     import fcntl
-except ImportError:
-    fcntl = None
 
-try:
-    from _thread import get_ident
+    _FCNTL_EXISTS = True
```

Comment:
It may be nice to add a note about the platforms which `fcntl` is expected to be available.